src/main.rs

```diff
@@ -23,7 +23,7 @@ mod xwrap;
 use crate::xwrap::Display;
 
 fn usage(progname: &str, opts: getopts::Options) {
-    let brief = format!("Usage: {progname} [options] [file]");
+    let brief = format!("Usage: {progname} [options] [file]\nNote: use - as [file] for stdout");
```

Comment:
I don't think this is the right place to document this, the help text should probably be reworked a bit to accommodate this. Ideally, the `[file]` positonal argument should have its own entry above the options section describing its behavior (both normal and special).
Also, since you're changing the help text, you have to update the readme to match.